kommunicate/src/main/java/io/kommunicate/utils/KmUtils.java

```diff
@@ -75,6 +73,13 @@ public static void setGradientStrokeColor(View view, int width, int color) {
         gradientDrawable.setStroke(width, color);
     }
 
+    public static void setDrawable(TextView textView, int drawableRes, int color)
```

Comment:
Method name "setDrawable" does not specify what exactly it does, name it in a way which tells what exactly this method does, so it can be used later on in other places.